pkg/cmd/openshift-controller-manager/conversion.go

```diff
@@ -164,23 +203,16 @@ func getPluginConfigObj(pluginConfig map[string]*configapi.AdmissionPluginConfig
 }
 
 // getPluginConfigYAML gives the byte content of the config for a given plugin
-func getPluginConfigYAML(pluginConfig map[string]*configapi.AdmissionPluginConfig, pluginName string, scheme *runtime.Scheme) ([]byte, error) {
+func getPluginConfigYAML(pluginConfig map[string]*legacyconfigv1.AdmissionPluginConfig, pluginName string, scheme *runtime.Scheme) ([]byte, error) {
 	// Check whether a config is specified for this plugin. If not, default to the
 	// global plugin config file (if any).
 	cfg, hasConfig := pluginConfig[pluginName]
 	if !hasConfig {
 		return nil, nil
 	}
-	obj := cfg.Configuration
-	if obj == nil {
+	if len(cfg.Configuration.Raw) == 0 {
```

Comment:
if cfg !=nil ?

Reply:
> if cfg !=nil ?

shouldn't be nil post validation.  The map presence check is above